tictactoe.gemspec

```diff
@@ -1,14 +1,15 @@
 Gem::Specification.new do |s|
   s.name = "tictactoe-randall"  # i.e. visualruby.  This name will show up in the gem list.
-  s.version = "0.0.2"  # i.e. (major,non-backwards compatable).(backwards compatable).(bugfix)
+  s.version = "0.0.3"  # i.e. (major,non-backwards compatable).(backwards compatable).(bugfix)
 	s.add_dependency "vrlib", ">= 0.0.1"
 	s.add_dependency "gtk2", ">= 0.0.1"
 	s.add_dependency "require_all", ">= 0.0.1"
+	s.licenses = ['MIT']
```

Comment:
This change should take effect when you accept my code.  When you push the gem, it will show MIT licence.
E 

      From: Randall Reed <notifications@github.com>
 To: randallreedjr/tictactoe_vr <tictactoe_vr@noreply.github.com> 
Cc: Eric Cunningham <Beagle4321_2000@yahoo.com>; Author <author@noreply.github.com>
 Sent: Saturday, May 21, 2016 1:31 PM
 Subject: Re: [randallreedjr/tictactoe_vr] fixed command line executable. (#2)
   
In tictactoe.gemspec:>  	s.add_dependency "vrlib", ">= 0.0.1"
>  	s.add_dependency "gtk2", ">= 0.0.1"
>  	s.add_dependency "require_all", ">= 0.0.1"
> +	s.licenses = ['MIT']
Can you also add a copy of the MIT license to the repo?—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub